vipps-checkout-how-it-works-shipping.md

```diff
@@ -1,12 +1,14 @@
 <!-- START_METADATA
 ---
-title: How it works(shipping)
+title: How it works for shipping
```

Comment:
Maybe Shipping: How it works (?)

Reply:
I was trying to avoid the colon to make it more like a sentence or a title. Any other ideas? If you guys don't like it, we can just put it back. I'll have to try and remember because I'm trying to make all the APIs match each other.

Reply:
hmm, its just that the "for" sound a bit odd in this case. Doesnt sound like a real sentence

Reply:
I like the "How it works: Shipping" , it is a title, so maybe not so bad?

Reply:
`How it works - Shipping` - if colon is the problem?

Reply:
I'll revert that, but use quotes instead of the parentheses.

Reply:
done

We should probably put quotes around all the titles to avoid problems with the ':' in the future. :-)